recipes/ioos_metrics/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "ioos_metrics" %}
+{% set version = "0.1.0.dev0" %}
```

Comment:
This one is an exception in CFEP-5 b/c it will be in dev mode for quite some time as it is in a state of flux due to the various services the package hits when requesting data.